secretmanager/api/SecretManager.Samples/UpdateSecretWithAlias.cs

```diff
@@ -0,0 +1,46 @@
+
```

Comment:
Do we need these extra newlines at the top of the file?

Reply:
Nope, my bad